SwiftWisdom/Core/StandardLibrary/Numbers/Mathable.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Mathable.swift
+//  SwiftWisdom
+//
+//  Created by Paul Rolfe on 2/12/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public protocol Mathable: Comparable {
```

Comment:
in researching this I realized that there is a protocol called IntegerArithmeticType.
https://developer.apple.com/library/tvos/documentation/Swift/Reference/Swift_IntegerArithmeticType_Protocol/index.html#//apple_ref/swift/intfm/IntegerArithmeticType/s:FPSs21IntegerArithmeticType8toIntMaxuRq_S__Fq_FT_VSs5Int64

this implements the operators below. Should Mathable simply inherit from this protocol?